src/libpsl-native/src/setdate.h

```diff
@@ -17,6 +17,6 @@ typedef struct setDateInfo
     int32_t DST;
 } SetDateInfo;
 
-int32_t SetDate(const SetDateInfo &info);
+int32_t SetDate(const SetDateInfo info);
 
```

Comment:
It's a C API, it can't be passed by reference (it's a compiler feature, not a standard, and shouldn't have been counted on to work). This is a pretty small struct of ints and is only used by `Set-Date`, I see no point in optimizing this by passing a pointer and dealing with further marshaling changes. Why worry about 8 ints on the stack once in a blue moon?